index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header-container">
+            <nav class="nav-item1"><a class="Mylogo" href="">My Logo</a></nav>
+            <nav class="nav-item2"></nav>
+            <nav class="nav-item3">
+                <img class="toggle" src="images/toggle.png" alt="">
+            </nav>
+            <div id="myNav" class="overlay">
+                <a class="closebtn"> &times; </a>
+                <div class="overlay-content">
+                    <a class="pop-nav" href="#portfolio">Portfolio</a>
+                    <a class="pop-nav" href="#about">About</a>
+                    <a class="pop-nav" href="#contact">Contact</a>
+                </div>
+            </div>
+        </div>
+        <nav class="web-nav">
+            <div class="logo">
+                <a class="Mylogo" href="">My Logo</a>
+            </div>
+            <ul>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="content">
+        <h1 class="intro">I'm Josue Rushanika Nice to see you!</h1>
+        <p class="intro1">
+            I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <h3 class="intro2">LET'S CONNECT</h3>
+        <ul class="social">
+            <li class="social-icon twitter">
+                
+                <a href="#"><img src="images/twitter.png" alt="twitter"></a>
+            </li>
+            <li class="social-icon linkedin">
+                <a href="#"><img src="images/Linkedin.png" alt="linkedin"></a>
+            </li>
+            <li class="social-icon">
+                <a href="#"><img src="images/m.png" alt="m"></a> 
+            </li>
+            <li class="social-icon github">
+                <a href="#"><img src="images/github.png" alt="github"></a>
+            </li>
+            <li class="social-icon fingers">
+                <a href="#"><img src="images/fingers.png" alt="fingers"></a>
+            </li>
+        </ul>
```

Comment:
- [Optional] Kindly, add your social media profile `URL` instead of using `#` because the `<a>` tag will let you connect your social networks with the icons. 👍 👍